src/Altinn.Register/test/Altinn.Register.Tests/IntegrationTests/PartiesControllerTests.cs

```
@@ -398,4 +399,176 @@ public async Task PostPartyNamesLookup_PartialInvalidInput_OK()
         Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         actualResult.Should().BeEquivalentTo(expectedResult);
     }
+
+    /// <summary>
+    /// Tests the PostPartyNamesLookup with invalid input and verifies that the appropriate error response is returned.
+    /// </summary>
+    /// <param name="socialSecurityNumbers">The Social Security Numbers.</param>
+    /// <param name="nameComponentOption">Specifies whether to include or exclude name components</param>
+    [Theory]
+    [MemberData(nameof(GetPartyLookupInvalidTestData))]
+    public async Task PostPartyNamesLookup_InvalidInput_BadRequest(string[] socialSecurityNumbers, string nameComponentOption)
+    {
+        // Arrange
+        PartyNamesLookup queryBody = new()
+        {
+            Parties = socialSecurityNumbers.Select(ssn => new PartyLookup { Ssn = ssn }).ToList()
+        };
+
+        int sblEndpointInvoked = 0;
+        HttpRequestMessage sblRequest = null;
+        DelegatingHandlerStub messageHandler = new((request, cancellationToken) =>
+        {
+            sblRequest = request;
+            sblEndpointInvoked++;
+
+            // Simulate an error response for invalid input
+            return Task.FromResult(new HttpResponseMessage(HttpStatusCode.BadRequest));
```

This message handler is for simulating the response from SBLBridge. In this test the the call chain should never get that far. The invalid input is detected long before a call to SBLBridge is being made. 

If we want to verify this properly we should avoid returning BadRequest here as that could confuse the intention of the test. Especially the comment. Could we return something else or maybe throw an exception instead? Remove the comment.

Another options could be to simply just not provide a message handler.